src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -28,60 +25,59 @@ const defaultProps = {
     },
 };
 
-class OnfidoStep extends React.Component {
-    /**
-     * @returns {boolean|*}
-     */
-    canShowOnfido() {
-        return this.props.walletOnfidoData.hasAcceptedPrivacyPolicy && !this.props.walletOnfidoData.isLoading && !this.props.walletOnfidoData.error && this.props.walletOnfidoData.sdkToken;
-    }
+function OnfidoStep({walletOnfidoData}) {
+    const {translate} = useLocalize();
+    
+    const shouldShowOnfido = walletOnfidoData.hasAcceptedPrivacyPolicy && !walletOnfidoData.isLoading && !walletOnfidoData.error && walletOnfidoData.sdkToken;
+
+    const goBack = useCallback(() => {
+        Wallet.updateCurrentStep(CONST.WALLET.STEP.ADDITIONAL_DETAILS);
+    }, []);
+
+    const reportError = useCallback(() => {
+        Growl.error(translate('onfidoStep.genericError'), 10_000);
```

Comment:
@adhorodyski, looks like a small mistake.
```suggestion
        Growl.error(translate('onfidoStep.genericError'), 10000);
```

Reply:
The underscore separator is a valid syntax, but do you say it's not widely used here? I'll update accordingly then!

Reply:
It looks interesting. I've never ever seen this before 😅
However, we don't use this syntax within the project.

<img width="433" alt="Screenshot 2023-09-20 at 10 50 59" src="https://github.com/Expensify/App/assets/57314631/0756dd2b-07ac-4591-a959-356cbda7b059">

Reply:
This makes sense, thanks for catching it :) I'll update shortly.